Source/Scene/Expression.js

```diff
@@ -343,6 +369,34 @@ define([
         return feature.getProperty(this._value);
     };
 
+    function checkFeature (ast) {
+        return (ast._value === 'feature');
+    }
+
+    // PERFORMANCE_IDEA: Determine if parent property needs to be computed before runtime
```

Comment:
OK. I will say it is currently not allowed, but I will add this feature 'Later' on the roadmap.